CHANGELOG

```diff
@@ -1,3 +1,8 @@
+* Tue Jul 07 2020 Kendall Moore <kendall.moore@onyxpoint.com> - 0.4.1-0
```

Comment:
We'll need to bump the module version in metadata.json before release.  I suggest 0.5.0 (details in manifests/init.pp)